include/ginkgo/core/solver/lower_trs.hpp

```diff
@@ -116,6 +116,15 @@ class LowerTrs : public EnableLinOp<LowerTrs<ValueType, IndexType>>,
          * (false) or should it assume they are 1.0 (true)?
          */
         bool GKO_FACTORY_PARAMETER_SCALAR(unit_diagonal, false);
+
+        /**
+         * Select the implementation which is supposed to be used for
+         * the triangular solver. This only matters for the Cuda
+         * executor where the choice is between the Ginkgo and the
+         * Cusparse implementation. Default is Ginkgo.
+         */
+        trisolve_algorithm GKO_FACTORY_PARAMETER_SCALAR(
+            algorithm, trisolve_algorithm::ginkgo);
```

Comment:
I think the cuSPARSE algorithm is more robust, especially in CUDA >= 11.3, so I would use that one by default.